src/Generator/PropertyStatementsGenerator.php

```diff
@@ -38,12 +39,14 @@ public function generate(PropertyMapping $propertyMapping): array
 
         $fieldValueVariable = $variableRegistry->getFieldValueVariable($propertyMapping);
 
-        if ($propertyMapping->hasCustomTransformer()) {
-            $output = $this->customTransformerExtractor->extract($propertyMapping->transformer, $fieldValueVariable, $variableRegistry->getSourceInput());
+        $transformer = $propertyMapping->getTransformer();
+
+        if (is_string($transformer)) {
```

Comment:
this is kinda "fragile" and gonna be refactored at some point (I'm wondering if I cannot make "custom transformers extractor" implement the main `TranformeInterface`)